clients/cli/src/main/java/org/apache/gravitino/cli/Main.java

```diff
@@ -72,7 +73,7 @@ protected static String resolveCommand(CommandLine line) {
     if (args.length == 2) {
       String action = args[1];
```

Comment:
Why not just do `toLowerCase(Locale.ROOT);` here? Then there is no need to modify `isValidCommand` and you can just `return action`.

Reply:
I think you also want to use `Locale.ENGLISH` as the commands are in English, using Locale.ROOT may give unexpected results.

Reply:
fix